android/build.gradle

```diff
@@ -12,12 +12,10 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:7.2.1'
-        classpath 'com.newrelic.agent.android:agent-gradle-plugin:6.10.0'
     }
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'newrelic'
```

Comment:
@ndesai-newrelic `com.newrelic.agent.android:agent-gradle-plugin:6.11.0` was causing issues with the plugin:
```
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring root project 'android'.
> Transforms with scopes '[EXTERNAL_LIBRARIES, SUB_PROJECTS]' cannot be applied to library projects.
```

I removed it from the project and the plugin was still working fine. I think it might be okay since we have users install this in their `app/build.gradle` but would like a second opinion.